pkg/minishift/openshift/version/openshift_versions.go

```diff
@@ -55,6 +56,25 @@ func GetOpenshiftVersion(host *host.Host) (string, error) {
 	return dockerCommander.Exec(" ", minishiftConstants.OpenshiftContainerName, "openshift", "version")
 }
 
+func GetOpenshiftVersionWithoutK8sAndEtcd(sshCommander provision.SSHCommander) (string, error) {
+	dockerCommander := docker.NewVmDockerCommander(sshCommander)
+	versionInfo, err := dockerCommander.Exec(" ", minishiftConstants.OpenshiftContainerName, "openshift", "version")
+	if err != nil {
+		return "", err
+	}
```

Comment:
@praveenkumar I do not think we should run `dockerCommander.Exec(" ", minishiftConstants.OpenshiftContainerName, "openshift", "version") ` in two functions, when the command is exactly the same. I would suggest we reuse GetOpenshiftVersion()  then parse the output to just get the OpenShift version information here. 

Also it might be a good idea to rename `GetOpenshiftVersion` to `GetOpenshiftClusterVersion` and `GetOpenshiftVersionWithoutK8sAndEtcd` to `GetOpenshiftVersion`

Reply:
@LalatenduMohanty If you see the signature of both function then it different. If you think we should call `GetOpenshiftVersion` from `GetOpenshiftVersionWithoutK8sAndEtcd` this is not possible atm. 

```
GetOpenshiftVersion(host *host.Host) (string, error)
GetOpenshiftVersionWithoutK8sAndEtcd(sshCommander provision.SSHCommander) (string, error)
```

@LalatenduMohanty @gbraad I don't think to execute a single command in both the function have any issue. I am also netural about naming part, let me know if you think we should change then will change it.